backend/README.md

```diff
@@ -1,26 +1,80 @@
-# RESTful API - .NET CORE
+# REST API - .NET CORE
 
-# Setup
+The PIMS API provides an interface to interact with the configured datasource.
 
-Create a .env file and populate it with this.
+# Configuration
+
+The current environment is initialized through the environment variable `ASPNETCORE_ENVIRONMENT`.
+
+When running the solution locally it applies the configuration setting in the following order;
+
+1. appsettings.json
+2. appsettings.`[environment]`.json
+3. UserSecrets `(if environment=Development)`
+4. Environment Variables
+
+To run the solution with docker-compose create a .env file and populate it with the following;
 
 ```
 ASPNETCORE_ENVIRONMENT=Development
-ConnectionStrings__GeoSpatial=Host=host.docker.internal;Port=5433;Username=sa;Password=[password];Database=geospatial;
-Keycloak__Secret=[secret]
+ASPNETCORE_FORWARDEDHEADERS_ENABLED=true
+DB_PASSWORD=<password>
+Keycloak__Secret=<secret>
 ```
 
+The `DB_PASSWORD` should be the same value used in the database `.env` file.
+The `Keycloak__Secret` should be the value provided by KeyCloak. Currently this value can remain blank.
+
 ## Secret Management
 
-To keep private keys and user secrets out of source code use the **user-secrets** management tool.
+If you want to keep private keys and user secrets out of source code use the **user-secrets** management tool.
+
+> `dotnet user-secrets init`
+
+> `dotnet user-secrets set "ConnectionStrings:GeoSpatial" "Host=host.docker.internal;Port=5433;Username=sa;Password=[password];Database=geospatial;"`
```

Comment:
Could you please change the above connection string to match SQL Server format? I believe MSSQL doesn't support `Host` like postgres